src/DataEventManager.php

```diff
@@ -48,17 +49,24 @@ public function sync(): ?ResponseInterface
             return null;
         }
 
-        return $this->client->request(
-            'POST',
-            $_ENV['DEVREL_DATA_PIPELINE_ENDPOINT'] . '/data/' . $this->projectName . '/batch',
-            [
-                'headers' => [
-                    'Accept' => 'application/json',
-                ],
-                'auth_bearer' => $_ENV['DEVREL_DATA_PIPELINE_TOKEN'],
-                'json' => $this->processEventDataList(),
-            ]
-        );
+        try {
+            return $this->client->request(
+                'POST',
+                $_ENV['DEVREL_DATA_PIPELINE_ENDPOINT'] . '/data/' . $this->projectName . '/batch',
+                [
+                    'timeout' => 1.0,
+                    'headers' => [
+                        'Accept' => 'application/json',
+                    ],
+                    'auth_bearer' => $_ENV['DEVREL_DATA_PIPELINE_TOKEN'],
+                    'json' => $this->processEventDataList(),
+                ]
+            );
+        } catch (TransportExceptionInterface $e) {
+            // ...
```

Comment:
next step would be to add a log.warning with the corresponding message 
but ok, we avoid HTTP 500 ;)